css/styles.css

```diff
@@ -145,12 +158,14 @@
 .ab-column > ul > li label {
     cursor: default;
     display: inline-block;
-    width: 286px;
+    line-height: 24px;
+    max-width: 286px;
 }
 .ab-column > ul > li label span {
     color: #8e959c;
     display: block;
     font-size: 12px;
+    line-height: 16px;
```

Comment:
In addition to line 161 this actually is a bit redundant but was restored from 2d193ccc36be60025bd5d8d82f6238b0203b5ffb.
Changing the line-height seems not appropriate right now.